MBBSEmu/Memory/MemoryCore.cs

```diff
@@ -506,6 +506,28 @@ public void SetArray(ushort segment, ushort offset, ReadOnlySpan<byte> array)
         public void SetArray(string variableName, ReadOnlySpan<byte> value) =>
             SetArray(GetVariablePointer(variableName), value);
 
+        /// <summary>
+        ///     Writes the specified byte the specified number of times starting at the specified pointer
+        /// </summary>
+        /// <param name="segment"></param>
+        /// <param name="offset"></param>
+        /// <param name="count"></param>
+        /// <param name="value"></param>
+        public void SetArray(ushort segment, ushort offset, ushort count, byte value)
+        {
+            for (var i = 0; i < count; i++)
```

Comment:
Can probably use some fancy `Span` methods to achieve this I would think

Reply:
Resolved using `Array.FIll()`